content-resources/src/main/java/flyway/oskari/V1_43_4__register_download_basket_bundle.java

```diff
@@ -0,0 +1,24 @@
+package flyway.downloadbasket;
+
+import java.sql.Connection;
+
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration;
+
+import fi.nls.oskari.db.BundleHelper;
+import fi.nls.oskari.domain.map.view.Bundle;
+
+public class V1_43_4__register_download_basket_bundle implements JdbcMigration{
+
+	private static final String NAMESPACE = "hsy";
```

Comment:
There's a "download-basket" bundle under hsy AND under framework namespaces. What's the difference between these two? Should this be for the one under framework?

Reply:
Yes, that's correct.